### PR TITLE
[ADD] web_widget_date_interval

### PR DESCRIPTION
**Pull request by hbrunn**
This is a widget for date intervals, currently only implemented for search views (the form part will be a migration of https://github.com/OCA/web/pull/575).

For testing, open the users list on runbot and play with the two extra dropdowns allowing you to filter for some intervals on the fields `create_date` and `last_login`

### CODE REVIEW

**Comment by ovnicraft** [comment]
@hbrunn any updates ?

**Comment by hbrunn** [comment]
not at the moment, but the addon is not dead. If you prefer, I can close this until I start working on it again. It's already useful for me, so I figured it might be for others as well, that's why I kept the PR here

**Comment by ovnicraft** [comment]
@hbrunn so its usable? 
We can review and merge it, so in next pr you can improve it.

**Comment by hbrunn** [comment]
well, usable is a big word. For now, you can add something like
screenshot_2019-01-08_17-01-45
to a search view, which allows you to cycle through calendar (iso) weeks of whatever date{,time} field you configured quickly. But it's not feature complete, so I think it will be quite a bummer for people to see this merged but not be able to do a lot of stuff, like having a form widget for date intervals

**Comment by ovnicraft** [comment]
Ok, so wait until you have time to end it 👍

File: web_widget_date_interval/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Date interval widget",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline web_widget_date_interval/__manifest__.py:7]: add `website` [like](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor change

**Pull request closed without merging**